### ISSUE TITLE
get_digital_currency_daily() market = USD

### ISSUE THREAD

**Issue by Ham-Merhead** [RomelTorres/alpha_vantage#33]
One small issue using "get_digital_currency_daily()". The result is returned in two currencies, market and USD, OK. But if I set the market to USD I get duplicate Pandas labels.

So indexing a row ( row['. close (USD)'] ) results in a two element series for USD, and a scalar for other markets.

The duplicate data comes from AV, but is not really an issue for them as they present data as a structured list and parsing would normally take the first match. However inserting all this data into a table (Pandas) results in duplicate keys. Indexing the table gets two matches and returns a list of two values rather than the single USD value we get from other markets.

Test case:
[usd.zip](https://github.com/RomelTorres/alpha_vantage/files/1552627/usd.zip)

This prints:
For GBP price returns <class 'numpy.float64'>
For USD price returns <class 'pandas.core.series.Series'>

**Comment by RomelTorres** [comment]
Hi @Ham-Merhead, I have been super busy and yesterday I just had time for some other important fixes. I will try to have a look over it and fix it over the weekend.

Best Regards Romel.

**Comment by Ham-Merhead** [comment]
Got it. Great, thank you.

In my code I changed the column keys to the full AV labels (e.g. '4b. close (USD)' or '4. close'), removed my workaround, and everything just worked :)   Always a nice experience.

I think your choice of solution is the sustainable one, but I would suggest a clear statement someplace. As newbies will assume for example 'close', and existing users will expect no change. Also I notice the index changed from a string (the date) to an int64 - no problem just different.

I tried:
get_digital_currency_daily()    with BTC and two markets USD and GBP  
get_daily_adjusted()      with SPY
